transport-native-io_uring/src/main/java/io/netty/channel/uring/AbstractIOUringServerChannel.java

```diff
@@ -69,12 +69,10 @@ public AbstractIOUringChannel getChannel() {
         return this;
     }
 
-    abstract Channel newChildChannel(int fd, byte[] address, int offset, int len) throws Exception;
+    abstract Channel newChildChannel(
+            int fd, long acceptedAddressMemoryAddress, long acceptedAddressLengthMemoryAddress) throws Exception;
 
     final class UringServerChannelUnsafe extends AbstractIOUringChannel.AbstractUringUnsafe {
-        // Will hold the remote address after accept4(...) was successful.
-        // We need 24 bytes for the address as maximum
-        private final byte[] acceptedAddress = new byte[24];
```

Comment:
I think I need that for SQPOLL for 5.8/5.9 as accept op is not supported

Reply:
If you need it we can add it again for the use-case.